### ISSUE TITLE
Progress bar misaligned in IE11

### ISSUE THREAD

**Issue by lalitha-jonnalagadda** [18F/crt-portal#165]
Progress bars and sub headers are misaligned when using IE11. Please see the mock up below for details. 

 ![image.png](https://images.zenhubusercontent.com/5b34455c5b51b10bd4dd59d6/4212e630-eb7e-40c2-9edc-7eba0dbd3c70)

**Comment by el-mapache** [comment]
This issue has a simple fix but was a bit tricky to diagnose! IE 11 has many bugs in its implementation of `flexbox`. A comprehensive list of `flexbox `related bugs can be found at the [handy flexbugs repo](https://github.com/philipwalton/flexbugs).

*NOTE*: In the rest of this comment I will be referring to the _property_ `flex`. This should not be confused with the `display` property value of `flex`. Confusing, I know. 😢 🐼 

### The issue

Basically, IE defaults the `flex` property to `0 0 auto`, which translates to a `flex-grow` and `flex-shrink` of 0. In plain english, the element isn't told to take up all of the available space allotted to it, particularly in the case of the `flex-grow` property.

Because the page's `body` element, which is acting as the global `flex` container to the page, has its `flex-direction` property set to `column` the `flex-grow` attributes of the `body`'s direct child elements are applied to the `height` of the container.

Now, given the following:

* The `body` defines a `flex-direction: column`, meaning direct children will be arrayed vertically
* The `header` element is a direct child of the `body` element
* `flex-grow` is set to 0 by default in IE

The `header` element will therefore *not* expand vertically to take up all available space, allowing the `main` element (which is also a direct child of the `body`!) to occupy space that should be filled by the `header` element.

The solution was to add a `flex-child` utility class, and apply that to the `header`. The flex-child class explicitly sets `flex: 1  0 auto`, telling the `header` element to occupy all the available space, given it's computed height.

**Comment by blaughman** [comment]
The progress bar now looks as it should.
